src/config.rs

```diff
@@ -224,10 +224,14 @@ impl From<ConfigUpstream> for PolicyStoreConfig {
     ) -> Self {
         Self {
             rpc: PolicyStoreRpcConfig { endpoint: rpc },
-            grpc: YellowstoneConfig {
+            grpc: YellowstoneGrpcConfig {
                 endpoint,
                 x_token,
                 timeout: 60,
+                commitment_level: Some(yellowstone_vixen::CommitmentLevel::Finalized),
```

Comment:
```suggestion
                commitment_level: Some(yellowstone_vixen::CommitmentLevel::Confirmed),
```
I think `Finalized` is too much time  ~12-14 seconds to receive updates, do we think this delay can impact some users?

Reply:
I'm looking at `yellowstone_shield_store` and apparently it's using `Confirmed`. https://github.com/rpcpool/yellowstone-shield/blob/3b377697c25b3acebd8bce7bd88d273127a0c664/store/src/lib.rs#L430